eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/DPState.hs

```diff
@@ -163,6 +164,12 @@ instance CC.Crypto crypto => FromSharedCBOR (PState crypto) where
     _retiring <- fromSharedPlusLensCBOR (toMemptyLens _1 id)
     pure PState {_pParams, _fPParams, _retiring}
 
+instance
+  ( CC.Crypto crypto
+  , FromSharedCBOR (PState crypto)
+  ) => FromCBOR (PState crypto) where
+  fromCBOR = evalStateT fromSharedPlusCBOR mempty
```

Comment:
We can fit it into one line and use `fromNotSharedCBOR`
```suggestion
instance (CC.Crypto crypto, FromSharedCBOR (PState crypto)) => FromCBOR (PState crypto)
  fromCBOR = fromNotSharedCBOR
```

Reply:
Fixed!